src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebUserGroupRouteConfig.kt

```diff
@@ -14,6 +14,11 @@ object WebUserGroupRouteConfig : RouteConfig
                     .json()
                     .secure(usersManage)
                     .transform(),
+            WebEndpoint("/user-groups/report-readers/:report/",
+                    UserGroupController::class, "getScopedReportReaders")
+                    .json()
+                    .secure(usersManage + setOf("*/reports.read"))
```

Comment:
The reason I thought this required more permissions is because it exposes a report name as existent. The global `/users/report-readers/` doesn't expose any report info, indeed it would be valid even if no reports existed!